commands/sandbox.go

```diff
@@ -0,0 +1,370 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const NODE_VERSION = "14.16.0"
```

Comment:
Is there a reason for this specific version? While 14.x is still supported, it looks like 16.x is the current LTS release. 

Looks like there are more recent 14.x releases as well: https://nodejs.org/download/release/latest-v14.x/

Reply:
That's the version that is normally baked into an installed version of `nim`.   Yes, it is not current LTS and should be updated.   Since we standardized on it a while back, it is the version I develop on and the version that our GitHub actions run tests on.   I know of no reason why switching to current LTS would break anything.   But, I'd prefer to spin off a task to update `nim` to the later version (should be fine, just needs some testing, including updating the GitHub actions) rather than making the change in this PR.    But, your call.